src/js/domain/LottoNumber.js

```diff
@@ -27,6 +27,11 @@ class LottoNumber {
     this.#value = validatedLottoNumber;
   }
 
+  static of(value) {
+    LottoNumber.validateLottoNumber(value);
+    return instances[Number(value)];
+  }
```

Comment:
```suggestion
  static of(value) {
    return new LottoNumber(value);
  }
```
이렇게 하면 자연스럽게 유효성 검사도 되겠죠!?